identity/webapp/copy.tsx

```diff
@@ -32,9 +32,20 @@ export const ValidatedSuccessText: FC<ValidatedSuccessTextProps> = ({
           library.
         </p>
         <p>
-          If you want to access subscription databases, e-journals and e-books,
-          you need to bring a form of personal identification (ID) and proof of
-          address to the admissions desk in the library.
+          To complete your membership and access subscription databases,
+          e-journals and e-books, you’ll need to bring a form of photo
+          identification (ID) and proof of your address to our admissions desk
+          when you visit.
+        </p>
+        <p>
+          <strong>Reminder:</strong> you will need to email a form of personal
```

Comment:
Were we trying to keep references of emailing personal id out of text?

Reply:
Argh, thanks for catching

Reply:
Fixed now.